src/main/java/org/dependencytrack/policy/ComponentAgePolicyEvaluator.java

```diff
@@ -58,6 +59,9 @@ public PolicyCondition.Subject supportedSubject() {
      */
     @Override
     public List<PolicyConditionViolation> evaluate(final Policy policy, final Component component) {
+        if(super.extractSupportedConditions(policy).isEmpty()){
```

Comment:
Would it make sense to assign the result of `super.extractSupportedConditions` to a variable, so that it won't be called twice when this check returns `false`?